config/routes.rb

```diff
@@ -105,8 +105,26 @@
       delete :delete_learning_object, on: :collection
     end
 
-
   end
 
+  # API
+  namespace :api do
+    namespace :v1 do
+      resources :concepts, only: [:index]
+      resources :concept_weeks, only: [:index]
+      resources :concepts_learning_objects, only: [:index]
+      resources :courses, only: [:index]
+      resources :feedbacks, only: [:index]
+      resources :learning_objects, only: [:index]
+      resources :setups, only: [:index]
+      resources :user_to_lo_relations, only: [:index]
+      resources :users, only: [:index]
+      resources :weeks, only: [:index]
+
```

Comment:
Mozno sem prihodit aj Answers (co moze niekoho zaujimat) a Setups_Users (co sa v buducnosti bude musiet dorobit aby to bolo relevantne) tabulky